src/actions/receive-profile.js

```diff
@@ -1266,7 +1267,9 @@ export function retrieveProfileFromFile(
               throw new Error('Unable to parse the profile.');
             }
 
-            await dispatch(viewProfile(profile));
+            await withHistoryReplaceStateSync(() => {
```

Comment:
We don't need to `await` here because `withHistoryReplaceStateSync` is a sync method. We could possibly use the async one here but it doesn't seem like the good choice because that actually waits until the end of the symbolication and therefore waits even after the profile view is ready. That makes stuff a lot more error prone because it's going to still be in the replace state even after profile view is ready.

So, we can just remove this await from here and keep using `withHistoryReplaceStateSync` :)

Reply:
Okay, will do